signer/src/config/mod.rs

```diff
@@ -833,6 +667,55 @@ mod tests {
         ))
     }
 
+    #[test]
```

Comment:
Some additional e2e sanity-check tests for #513 -- this is all covered in the new tests in `serialization.rs`, however.